.prettierignore

```diff
@@ -3,4 +3,8 @@ playgrounds/
 shared-configs/
 web-apps/
 
+packages/genesys-spark-components/src/components/**/readme.md
+packages/genesys-spark-tokens/data/tokens.json
+packages/genesys-spark-visualizations/src/components/**/readme.md
```

Comment:
This change is linked to @MattCheely 's removal of `package/` from this file to get prettier to work in his IDE. We do not want to reformat auto generated files

Reply:
Do .prettierignore configs in the child packages not work for this as well? It's not clear to me what the expectations are for prettier in this nested package format.